### PR TITLE
Dynamic product instructions

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Make the services enabling guidelines dynamic based on the personal accounts affordences.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Login
- See your token 
- See that it lists the services you have available
- Click on them to see the commands to enable and disable them

## Issue / Card
Fixes https://github.com/canonical-web-and-design/base-squad/issues/708

### CODE REVIEW

**Comment by barrymcgee** [comment]
@anthonydillon Quite a few conflicts need resolving here ☝🏻

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 PORT=8001
-DEBUG=true
```
Comment (barrymcgee) [inline .env]: Debugging in production?

File: static/sass/styles.scss [hunk 2]
```diff
@@ -284,3 +284,10 @@ summary {
     }
   }
 }
+
+// Advantage section
+#enterprise-table { // sass-lint:disable-line no-ids
```
Comment (barrymcgee) [inline static/sass/styles.scss]: If we have control over this element, why add an id and not a class?

File: static/sass/styles.scss [hunk 3]
```diff
@@ -284,3 +284,10 @@ summary {
...
+  .p-table tr:target {
```
Comment (barrymcgee) [inline static/sass/styles.scss]: Won't a `<tr>` always be in a table pattern, overqualified selector?

File: templates/advantage/_table.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
+
+<div class="row">
+  <div class="table" id="enterprise-table">
+    <div class="row">
+      <div class="col-12 u-sv3" style="overflow-x: auto;">
+        <table class="p-table" style="width: 100%;">
```
Comment (barrymcgee) [inline templates/advantage/_table.html]: A `<table>` element is `width: 100%` by default in Vanilla.

File: templates/advantage/index.html [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+            </section>
+          </li>
+        </ul>
+      </aside>
+    </div>
+  </div>
+
+  {{ accounts }}
```
Comment (barrymcgee) [inline templates/advantage/index.html]: This is dumping everything into the view template:

Screenshot 2019-09-06 at 08 17 13

File: templates/advantage/index.html [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+              </div>
+            </span>
+          </span>
+        </li>
+        {% endif %}
+        {% if personal_account['affordances']['fips'] %}
+        <li class="p-inline-list__item">
+          <span class="p-contextual-menu">
+            <a href="#" class="p-contextual-menu__toggle" aria-controls="#menu-fips" aria-expanded="false" aria-haspopup="true">FIPS</a>
+            <span class="p-contextual-menu__dropdown" id="menu-fips" aria-hidden="true" aria-label="submenu" style="min-width: 20rem; right: -9rem; padding: 1rem;">
```
Comment (barrymcgee) [inline templates/advantage/index.html]: This inline style is duplicated thrice indicating it should be abstracted to a class.

**Comment by anthonydillon** [comment]
Moving this PR to https://github.com/canonical-web-and-design/ubuntu.com/pull/5771 but fixing your points @barrymcgee

File: static/sass/styles.scss (hunk 2 above)
Comment (anthonydillon) [reply static/sass/styles.scss]: Good point

File: static/sass/styles.scss (hunk 3 above)
Comment (anthonydillon) [reply static/sass/styles.scss]: Good point

File: templates/advantage/_table.html (hunk 4 above)
Comment (anthonydillon) [reply templates/advantage/_table.html]: It's gone

File: templates/advantage/index.html (hunk 5 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: Removed

**Comment by anthonydillon** [comment]
Superseeded by https://github.com/canonical-web-and-design/ubuntu.com/pull/5771

**Pull request closed without merging**